source/common/http/http1/codec_impl.h

```diff
@@ -254,6 +254,15 @@ class ConnectionImpl : public virtual Connection, protected Logger::Loggable<Log
     NoBodyData = 2,
   };
 
+  /**
+   * An inner dispatch call that executes the dispatching logic. While exception removal is in
+   * migration (#10878), this function may either throw an exception or return an error status.
+   * Exceptions are caught and translated to their corresponding statuses in the outer level
+   * dispatch.
+   * TODO(#10878): Remove this when exception removal is complete.
+   */
+  Http::Status innerDispatch(Buffer::Instance& data);
+
```

Comment:
nit: Consider moving this method back to the private section to minimize whitespace changes.